### DISCUSSION TITLE
我差点在工程里写出三种大小写的 「id」

### DISCUSSION THREAD

**Discussion by pnck** [flipped-aurora/gin-vue-admin#955] [General]
- `GVA_MODEL` 是没有 `json` `struct tag` 的，因此它被序列化成 json 的时候会有 `ID` 字段
- 自动化代码模板使用 `camelCase` ，所以不用 `GVA_MODEL` 的 model 我写了  ```  Id uint `json:"id" gorm:"primaryKey"` ```
- 而且，这工程里函数形参的大小写也是不统一的，你能搜到 `Id` 的入参

我看着接口返回来的 `Id id ID` 陷入了沉思…… 🤔

**Comment by snans** [comment]
确实，我们这种追求统一风格的人特别纠结。
